src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/DefaultOverridenBundlePathPass.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\Config\Resource\FileExistenceResource;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Registers the default overriding bundles paths.
+ */
+class DefaultOverridenBundlePathPass implements CompilerPassInterface
```

Comment:
```suggestion
class DefaultOverriddenBundlePathPass implements CompilerPassInterface
```

note, that the filename needs to be changed too ❗️